dspace/janitor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "DSpace",
+  "description": "DSpace is a turnkey institutional repository application.",
+  "icon": "https://janitor.technology/img/dspace.svg",
+  "docker": {
+    "image": "janitortechnology/dspace"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https",
+      "preview": true
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    }
```

Comment:
Nit: You probably want to add the xposed ports `4403 8000 8080 9876` here as well (otherwise, Janitor won't allocate instance ports for them, and users won't be able to access them).

Reply:
I think there are a few of those ports that are not necessary